util/Migrator/DbScripts/2024-09-03_00_DropOrganizationUserAccessAll_DefaultColumnValue.sql

```diff
@@ -0,0 +1,11 @@
+-- Finalise removal of OrganizationUser.AccessAll column
+-- Add default column value
+-- Sprocs already have default value for rollback purposes, this just supports dropping the column itself
+
+IF OBJECT_ID('[dbo].[DF_OrganizationUser_AccessAll]', 'D') IS NULL
+  AND COL_LENGTH('[dbo].[OrganizationUser]', 'AccessAll') IS NOT NULL
```

Comment:
@rkac-bw the equivalent migraiton for `Groups` didn't check for the existence of the column (only for the non-existence of the constraint), so it throws an error if run multiple times. Theoretically dbup shouldn't run a migration multiple times, but it is something we try to support. Do you want me to go back and fix that?

Compare: https://github.com/bitwarden/server/blob/main/util/Migrator/DbScripts/2024-09-02_00_DropGroupAccessAll_DefaultColumnValue.sql#L5